src/protagonist/Engine/Infrastructure/ServiceCollectionX.cs

```diff
@@ -164,4 +168,13 @@ public static IServiceCollection ConfigureHealthChecks(this IServiceCollection s
 
         return services;
     }
+
+    public static IServiceCollection AddTopicNotifiers(this IServiceCollection services)
+    {
+        services
+            .AddScoped<IBatchCompletedNotificationSender, BatchCompletedNotificationSender>()
+            .AddScoped<IBatchCompletedNotificationSender, BatchCompletedNotificationSender>();

```

Comment:
Duplicate dependency. Does this method need to be called from API too, for raising notifications after `/test`?